tests/utils/helpers.ts

```diff
@@ -0,0 +1,244 @@
+                  return false;
+               }
+            } else if (filter.offerOption.includes('-max')) {
+               // Slider setted maximum value(currently there is no one in app)
+               if ((offer[filter.offerKeyName] as number) > filter.value) {
+                  // When salary of job offer is greater than maximum setted value then array is incorrectly filtered
+                  return false;
+               }
+            }
+         } else if (
+            typeof offer[filter.offerKeyName] === 'string' &&
+            typeof filter.value === 'boolean'
+         ) {
+            // When checkbox is checked then compare offer data with setted to filter
+            //  (eg. When "Job Type" filter checkbox eg. "Contract" is checked then
+            //      offer.jobType should be equal to Contract, and only that offers should be displayed )
+            if (
+               filter.value &&
+               (offer[filter.offerKeyName] as string).toLowerCase() !==
+                  filter.offerOption.toLowerCase()
+            ) {
+               // Values doesn't match, offer list is incorrectly filtered
+               return false;
+            }
+         } else if (
+            typeof offer[filter.offerKeyName] === 'string' &&
+            typeof filter.value === 'string'
+         ) {
+            // Check input inputs agains proper filtering.
+            // Offerlist should display only that offers which includes provided input text(job title/location)
+            if (
+               !(offer[filter.offerKeyName] as string)
+                  .toLowerCase()
+                  .includes((filter.value as string).toLowerCase())
+            ) {
+               // Values doesn't match, offer list is incorrectly filtered
+               return false;
+            }
+         }
+      }
+   }
+   // Offer list is correctly filtered
+   return true;
+};
+
+export const checkIfFiltersAreReseted = async (): Promise<boolean> => {
+   const filters = screen.queryAllByRole('filters-section');
+   for (const filter of filters) {
+      const filterElements = filter.querySelectorAll('[data-testid="filter-option"]');
+      expect(filterElements.length).toBeGreaterThan(0);
+
+      for (const filterElement of filterElements) {
+         // Get single checkbox data
+         const dataBefore: FilterData = JSON.parse(
+            filterElement.getAttribute('data-test-option') as string,
+         );
+         // Each [boolean] filter element should set to false
+         if (typeof dataBefore.value === 'boolean' && dataBefore.value !== false) {
+            return false;
+         }
+         // Each [number] filter element should set to 0
+         if (typeof dataBefore.value === 'number' && dataBefore.value !== 0) {
+            return false;
+         }
+      }
+   }
+   return true;
+};
+
+export const setRandomFilters = async () => {
```

Comment:
**issue:** What filters? It's impossible to track, and maintain. Let's split it to `setRandomLocationFilter`, `setRandomJobTypeFilter`, etc.

Reply:
All filters under "filters-section". So all filters as we have only one filters section in app.
`
export const setRandomFilters = async () => {
   const filters = screen.queryAllByRole('filters-section');
   for (const filter of filters) {...}}
`

It's possible to track by adding console.log with currently failed testing component. And don't need to change anything here in order to extending filters in app (eg. new country filters, this function will automatically test them too). 

This was my idea, to be able to test new filters immediately after adding them to the application - without having to write new tests for each new type.

@Sumick, bearing the above in mind - should I leave the current solution or make the suggested changes?